Patch TutorialTriggerGroup

Vhostym opened the pull request:
Remove usage of deprecated OnLevelWasLoaded function from TutorialTriggerGroup.

Vhostym committed:
Patch TutorialTriggerGroup

Remove usage of deprecated OnLevelWasLoaded function from TutorialTriggerGroup.

ambid17 commented on Assets/Scripts/Level/TutorialTriggerGroup.cs:
```diff
@@ -30,6 +33,10 @@ void UpdateTriggers()
         gameUI = playerProgress.GetComponentInChildren<InGameUI>(true);
+        if (playerProgress == null || gameUI == null)
+        {
+            Debug.LogError("Unable to find playerProgress or gameUI");
```
probably add a return to prevent uncaught nullref below

Vhostym replied:
It doesn't change anything since null is a valid value for those elements so the rest of the code functions fine. But sure. It'll save a few CPU cycles.

Vhostym committed:
Don't populate Triggers if values not found.

Add Exit out of the UpdateTrigger function if either no playerprogress or no gameUI is found.
